src/test/java/com/softserveinc/ita/HomePageTest.java

```diff
@@ -53,6 +53,28 @@ public void verifyCategoryTransitionThroughDropdownTest(String title) {
                 .contains(title);
     }
 
+    @DataProvider
+    public Object[][] rozetkaSocialMedia() {
+        return new Object[][]{
+                {"Facebook"},
+                {"Telegram"}
+        };
+    }
+
+    @Test(dataProvider = "rozetkaSocialMedia")
+    public void verifySocialMediaLinks(String socialMediaName) {
+        homePage.openSocialMediaPage(socialMediaName);
+        var expectedUrl = SocialMedia.valueOf(socialMediaName.toUpperCase()).getSocialMediaLink();
```

Comment:
implement such interface:
```
public interface IUiTextEnum {

    static <T extends Enum<T> & IUiTextEnum> T fromValue(String name, Class<T> enumType) {
        EnumSet<T> all = allOf(enumType);

        for (T aEnum : all) {
            if (aEnum.name().equalsIgnoreCase(replace(name, SPACE, "_"))) {
                return aEnum;
            }
        }
        throw new IllegalArgumentException("Unknown " + enumType.getSimpleName() + ": " + name);
    }
}
```

create method in enum SocialMedia after implementing interface above

```
public static SocialMedia getSocialMediaBy(@NonNull String name) {
        return fromValue(name, SocialMedia.class);
    }
```

Reply:
and after this you will be able replace this "SocialMedia.valueOf(socialMediaName.toUpperCase())" by "getSocialMediaBy(socialMediaName)"

Reply:
this method was in master already, social media links was made by Maks

Reply:
ok, probably create task on this and add my comments to description